client/app/components/AboutUsTemplate/index.js

```diff
@@ -65,37 +65,49 @@ const AboutUsTemplate = () => (
           name="Garth Froese"
           title="CEO"
           imgSrc="http://res.cloudinary.com/dyjo5a3ci/image/upload/v1514087314/idicia/garth-froese.jpg"
-          description="Very CEO wow guy ..."
+          description="Garth focuses on building bridges.Bridges between opportunities, people, and businesses. 
```

Comment:
Have you verified that this is rendering properly? You normally can't do a multiline string in Javascript without using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)